bin/test/i18n/test_metrics.rb

```diff
@@ -57,6 +57,7 @@
     end
 
     before do
+      described_class.unstub(:log_metric)
```

Comment:
I'd love to see a comment calling out where this is getting stubbed in the first place. Also, do we need to re-stub the method after this test?